### DISCUSSION TITLE
Episode 153: Pascal vs C vs Ada with Jonathon O'Connor

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#45] [Episode]
Episode Link: https://adspthepodcast.com/2023/10/27/Episode-153.html

Feel free to write comments or ask questions 🙂

**Comment by happygiraffe** [comment]
Possibly relevant to the Pascal vs C discussion: Brian Kernighan's paper [Why Pascal is Not My Favorite Programming Language](https://www.cs.virginia.edu/~evans/cs655/readings/bwk-on-pascal.html).
